app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -42,7 +42,7 @@
 import com.google.common.collect.Maps;
 
 public class DynamoSurveyDao implements SurveyDao {
```

Comment:
Survey elements are always retrieved as part of a survey. If we cache a survey, we cache the elements. The only place we query for them is in the query for a survey.